code/modules/reagents/chems/chems_herbal.dm

```diff
@@ -15,10 +15,16 @@
 /decl/material/liquid/antitoxins/ginseng
 	name               = "powdered ginseng"
 	uid                = "chem_antitoxins_herbal"
-	lore_text          = "Ginseng root has curative properties and encourages organ recovery after poisoning."
+	lore_text          = "Ginseng root has curative properties and encourages organ recovery and restoration of blood volume after poisoning or blood loss."
 	taste_description  = "bitter herbs"
 	antitoxin_strength = 0.35
 
+/decl/material/liquid/antitoxins/ginseng/affect_ingest(var/mob/living/M, var/removed, var/datum/reagents/holder)
+	. = ..()
+	if(M.has_trait(/decl/trait/metabolically_inert))
+		return
+	M.add_chemical_effect(CE_BLOODRESTORE, 8 * removed)
```

Comment:
this will mean ginseng tea and tincture don't apply it, no? or at least that it wouldn't be more effective